src/bp/core/database/tables/server-wide/messages.ts

```diff
@@ -0,0 +1,25 @@
+import { Table } from 'core/database/interfaces'
+
+export class MessagesTable extends Table {
+  name = 'web_messages'
```

Comment:
same for this one

Reply:
+ you might want to store the table name in an exportant constant  for reusability purposes. When we'll refact this, we might forget some cases where it can be used

Reply:
name is a property of the `Table` abstract class. So it has to be public and is used for logs and could be used for migrations.